scripts/visual-active-effects.mjs

```diff
@@ -36,17 +36,46 @@ export class VisualActiveEffects extends Application {
       // Set up the various text (intro and content).
       const desc = entry.effect.flags["dfreds-convenient-effects"]?.description;
       const data = entry.effect.flags[MODULE]?.data ?? {};
+      
+      // Get the effect rollData to populate enrichers within the descriptions.
+      let rollData = {};
+      try {
+        if (entry.effect.origin) {
+          let origin = await fromUuid(entry.effect.origin);
+          if (origin?.documentName === ActiveEffect.documentName) {
+            // Change the origin to the parent of the ActiveEffect - the originating item or actor.
+            origin = origin.parent;
+          }
+          if (origin?.pack) {
```

Comment:
This is fine to keep despite moving to `fromUuidSync` as it may retrieve a document that has been already fully loaded.